docs/sec_av_scatter.rst

```diff
@@ -155,35 +155,32 @@ using :func:`~dclab.kde.contours.get_quantile_levels`.
     # load the example dataset
     ds = dclab.new_dataset("data/example.rtdc")
     kde_instance = KernelDensityEstimator(ds)
-    X, Y, Z = kde_instance.get_raster(xax="area_um", yax="deform")
-    Z /= Z.max()
+
     quantiles = [.1, .5, .75]
-    levels = dclab.kde.contours.get_quantile_levels(density=Z,
-                                                    x=X,
-                                                    y=Y,
-                                                    xp=ds["area_um"],
-                                                    yp=ds["deform"],
-                                                    q=quantiles,
-                                                    )
+
+    contours, levels = kde_instance.get_contour_lines(xax="area_um",
+                                                      yax="deform",
+                                                      ret_levels=True,
+                                                      quantiles=quantiles)
+
+    linestyles = ["--", "--", "-"]
+    colors = ["b", "r", "g"]
 
     ax = plt.subplot(111, title="contour lines")
     sc = ax.scatter(ds["area_um"], ds["deform"], c="lightgray", marker=".", zorder=1)
-    cn = ax.contour(X, Y, Z,
-                    levels=levels,
-                    linestyles=["--", "-", "-"],
-                    colors=["blue", "blue", "darkblue"],
-                    linewidths=[2, 2, 3],
-                    zorder=2)
+
+    for i, (cnt, lvl) in enumerate(zip(contours, levels)):
+        ax.plot(cnt[0][:, 0], cnt[0][:, 1],
```

Comment:
Here you should also iterate over all contours in `cnt`, not only plot the first contour `cnt[0]`. There could be more than one contour line.